gradle/template.gradle.properties

```diff
@@ -73,7 +73,7 @@ systemProp.file.encoding=UTF-8
 # Set up gradle JVM defaults.
 #
 # We also open up internal compiler modules for spotless/ google java format.
-org.gradle.jvmargs=-Xmx2g -XX:ReservedCodeCacheSize=256m -XX:TieredStopAtLevel=1 -Dfile.encoding=UTF-8 -Dkotlin.daemon.jvm.options\="-Xmx2048M" -XX:+UseParallelGC -XX:ActiveProcessorCount=1 \
+org.gradle.jvmargs=-Xmx2g -XX:ReservedCodeCacheSize=256m -XX:TieredStopAtLevel=1 -Dfile.encoding=UTF-8 -Dkotlin.daemon.jvm.options="-Xmx4096M" -XX:+UseParallelGC -XX:ActiveProcessorCount=1 \
```

Comment:
does this NEED to be 4G? isn't this saying that the rest of the build is using Xmx2g and just for the UI compilation is needing 4GB?

Reply:
Since we allow people to disable the UI module build, I will keep it as it is, so that the memory configuration for building the project (without the new UI) is not increased.

Feel free to update the template to 4GB if it is better overall.